ThunderTable/TableViewController.swift

```diff
@@ -577,6 +577,47 @@ open class TableViewController: UITableViewController, UIContentSizeCategoryAdju
             set(indexPath: indexPath, selected: false)
         }
     }
+
+    override open func tableView(
+        _ tableView: UITableView,
+        heightForHeaderInSection section: Int
+    ) -> CGFloat {
+        guard let tableSection = data[section] as? DeclarativeSection else {
+            return super.tableView(tableView, heightForHeaderInSection: section)
```

Comment:
Have you checked that this doesn't crash... sometimes calling super in classes like this has given me issues in the past!

Reply:
I'd expect it to be defined in the super class (or at least the `UITableViewController`)? Though I know what you mean, I've seen that with `willDisplay...`